### PR TITLE
Adds cloth sheets

### PR DESCRIPTION
**Pull request by Kurfursten**
- Cloth sheets are actually in the game before, but this adds recipes to them
- There's a 30% chance they show up in the tool storage closets in aux tool storage, too
- You can rip up jumpsuits for more
- Useful for making gauze, rags, surgical masks, and white jumpsuits, gloves, caps.
- You can't rip up head jumpsuits
- You can't rip up a suit while it's still on your body

Stop bleeding in a pinch: rip up your suit today.

### CODE REVIEW

**Comment by 9600bauds** [comment]
This does allow you to get gloves extremely easy but honestly who gives a shit, nobody scans for fingerprints anyways.

**Comment by Probe1** [comment]
lots of people scan for fingerprints.

you probably want to do something like latex gloves and passing fingerprints through.

**Comment by Kurfursten** [comment]
The solution to gloves is the same that it has always been: they leave behind fibers, and you can scan the gloves themselves for prints. If you find white fibers, anyone making their own gloves falls under the eye of suspicion. If you find abandoned white gloves in maintenance or in disposals, scan them for prints.

Cloth doesn't transmit prints in the way that latex does.

**Comment by 9600bauds** [comment]
I don't know, the description for the white gloves says that they are "very fancy". You can't really make anything fancy or complex out of scraps of cloth and no tools.
Plus stuff like someone ripping up their jumpsuit and re-making it to turn it into white.

**Comment by ComicIronic** [comment]
I'm not sure about creating jumpsuits and gloves, especially doing so perfectly efficiently

**Comment by Probe1** [comment]
honestly if you stopped at gauze and rags it'd be great.  I forget molotovs are even in the game because no rags ever.

**Comment by Kurfursten** [comment]
I think there's a compelling case for removing gloves from the listing, but I'm not convinced removing jumpsuits is even on the table. Suitlords are on every map, and white jumpsuit is nominally (e.g.: to the examiner's eye) different than any job suits, and is apparent to any observer that a white suit has either been bought at a suitlord or hand-made, so it should set off warning bells (especially when you see a 'doctor or scientist' without colored epaulets and labcoat). This provides a good option to everyone caught in a plasma flood who needs to decontaminate, but doesn't want to go without pockets and ID for 2 minutes while they walk to the suitlord or washing machine.

**Comment by ComicIronic** [comment]
But that's an abuse of the system, really.
You shouldn't be able to get rid of plasma by tearing apart and tying back together your jumpsuit.

Especially since it makes no sense that a jumpsuit provides exactly the number of bolts required to make it.
Hell, being able to make a full jumpsuit at all is fishy: most people can't make anything beyond a toga with cloth.

**Comment by Kurfursten** [comment]
@ComicIronic You must be a more multifariously talented man than I; for I cannot make comfy chairs out of sheets of metal, even given several minutes to work.

**Comment by ComicIronic** [comment]
I am against a large part of the metal crafting system for the same reason.

**Comment by Kurfursten** [comment]
@ComicIronic I'm not a stubborn guy needlessly for the sake of being stubborn. If we have a toga in game or you have a toga sprite/can acquire one, I'll throw it in the PR. Then we can put the jumpsuit issue to bed.

**Comment by ComicIronic** [comment]
I'm still against the perfect conversion.

If you can't take apart an rwall in 15 steps without losing a sheet of plasteel for your trouble, you can't rip up a jumpsuit perfectly.

**Comment by Kurfursten** [comment]
>If you can't [the single most exceptional crafted thing in SS13, which no one ever makes outside of buildmode]

Not a useful standard of measure

**Comment by ComicIronic** [comment]
I don't think it works with a regular wall either.

And I have built rwalls in the past.

**Comment by FudgePucker** [comment]
I am on my phone so I can't really look at the code, but if it is a mat (like metal and glass) make it where the cost of a jumpsuit is more cloth than you get. Bamn, problem solved. I think cloth would be a neat inclusion and open up more possibilities, like hammocks that you can hang from the roof, bags, and a ton of other things. Maybe add hemp or cotton as a plant, but that's outside the scope of this PR. Awesome job.
@ComicIronic
God forbid we nerf plasma flooding, the thing that causes everyone to die besides the people wearing one of the two plasmaproof rigsuts or silicons. Being able to make jumpsuits isn't going to change much to the best way of crashing this station, WITH NO SURVIVORS.

File: code/modules/clothing/clothing.dm [hunk 1]
```diff
@@ -435,4 +436,22 @@ BLIND     // can't see anything
...
 
+/obj/item/clothing/under/proc/rip(mob/user)
+	user.visible_message("<span class='warning'>[user.name] is ripping apart the [src]!</span>", \
+			"<span class='warning'>You begin to rip apart the [src].</span>", \
+			"You hear an audible ripping.")
+	if(do_after(user,40))
+		new /obj/item/stack/sheet/cloth(user.loc,4)
+		qdel(src)
 
+/obj/item/clothing/under/verb/rip_up()
```
Comment (ComicIronic) [inline code/modules/clothing/clothing.dm]: You need to check usr.restrained() || usr.stat

**Comment by ComicIronic** [comment]
I suggest you pool the sheets - if crafting is going to happen, it's going to generate some unnecessary cost with the news and dels.

**Comment by Kurfursten** [comment]
@ComicIronic NFD?

**Comment by ComicIronic** [comment]
Not until the fact that you can rip up a jumpsuit and form it back together perfectly for an instant disguise or a contamination cleaning is fixed.

**Comment by Kurfursten** [comment]
@ComicIronic I don't think it would be a good disguise. It now has a unique new sprite that looks shitty and ripped up.

**Comment by Kurfursten** [comment]
An informal strawpoll at peak hours returned the following data. Of all respondents: 72% percent approved of ripping up jumpsuits as a means of acquiring cloth. Of those respondents who approved of ripping up jumpsuits at all, information about their opinions on how much cloth should be returned is as follows:
**Median**: 3 pieces
**Mean**: 3.125 pieces

As there is a decisive community opinion, I'll code it to those specifications.

**Comment by Kurfursten** [comment]
I don't think this idea is really "there" yet. It has some promise but there's just not enough stuff that can be made out of cloth, and moreover, not enough good sources of cloth aside from assaulting a Suitlord. Maybe in the future, a more expansive PR that includes cotton plants for botany. At this point, it just feels like a half-finished idea that was rushed out.

**Comment by FudgePucker** [comment]
@ComicIronic 
I have already suggested a solution, have it where ripping up cloth objects gives you less cloth than the object takes to make. So that you can't just take off your jumpsuit rip it up to get another jumpsuit, you need to also rip up another one.
@Kurfursten 
DON'T WORRY, I MIGHT SUCK AT MOST THINGS BUT I AM A OKAY IDEAS GUY.
Idea 1# rope. you can attach it to walls, meaning people can clip/hold on (Buckle) to so they don't, fly across the room when an airlock is opened, cross space without slipping, pull themselves across space lube (walk intent buckled in, you just pull yourself across.) Also allows for implications of pully systems, have a trip wire linked to a pully holding up a loony toons style anvil trap as a clown. Make diffrent knots, make a monkeys fist to attack security with, make a noose to hang the assistant who attacked you with a monkeys fist, make everyone uneasy when you take your 'speciest' antivox gimmick way too far.
Idea 2# Clothing. The outfit options are limitted, and most of the uniqe ones are in a vender only accessable to the clown and mime. Why not make people interact with the boatanist more by giving them a loom/sewing machine by their front desk? Just need some more outfits and uniforms and clothing. Even better, the boatanist can make more fibers than just cotton, mutating the cotton can make stronger fibers, meaning you can make stronger jumpsuits like sec has... only stronger. Hell why not after getting the poatancy high enough, getting enough space spider silk, and a human sacrifice, why not just let them make a super armor/rigsuit that is super light and strong?

**Comment by ComicIronic** [comment]
>I have already suggested a solution, have it where ripping up cloth objects gives you less cloth than the object takes to make. So that you can't just take off your jumpsuit rip it up to get another jumpsuit, you need to also rip up another one.

And that solution hasn't been implemented, despite me calling for it. That's why the PR didn't get NFD.

**Comment by Kurfursten** [comment]
It's not enough. The idea isn't developed fully. It shouldn't even be implemented without a real source (e.g.: botany, supply crates) but I don't have the time or motivation right now to develop the PR to that level.

**Pull request closed without merging**